zenoh-kuksa-provider/Cargo.toml

```diff
@@ -20,9 +23,11 @@ edition = "2021"
 clap = { version = "4.5.18", features = ["derive", "env"] }
 env_logger = "0.10.2"
 json5 = "0.4.1"
-kuksa = { git = "https://github.com/eclipse-kuksa/kuksa-databroker.git", branch = "feature/refactor_kuksa_crate", package = "kuksa" }
+kuksa-rust-sdk = "0.1.2"
```

Comment:
wanna use the latest version? or pinned on this version? 0.1.3 got released or use latest?